### PR TITLE
create WebIDL skeleton by fixing all WebIDL issues

### PR DESCRIPTION
**Pull request by danielpeintner**
Still misses prose/description of WebIDL interfaces, attributes and operation

### CODE REVIEW

**Comment by zolkis** [comment]
@danielpeintner Could you create a github.io page to be able to share a rendered version of the changes? (Of course I could also copy it to mine, but it would be nice if PRs would include a github.io link for html specs).

**Comment by danielpeintner** [comment]
Hi @zolkis!
Shall we use the htmldiff provided by W3C? (see https://services.w3.org/htmldiff?doc1=https%3A%2F%2Fw3c.github.io%2Fwot-scripting-api%2F&doc2=https%3A%2F%2Fraw.githubusercontent.com%2Fdanielpeintner%2Fwot-scripting-api%2Fmaster%2Findex.html)

**Comment by zolkis** [comment]
Perfect!

File: index.html [hunk 1]
```diff
@@ -501,32 +505,124 @@
...
+        <li>The <dfn>emitEvent()</dfn> method xyz.</li>
+        <li>The <dfn>onRetrieveProperty()</dfn> method xyz.</li>
+        <li>The <dfn>onUpdateProperty()</dfn> method xyz.</li>
+        <li>The <dfn>onInvokeAction()</dfn> method xyz.</li>
+        <li>The <dfn>onObserve()</dfn> method xyz.</li>
+      </ul>
+      
+      <section data-dfn-for="PropertyRequestHandler">
+        <h2><dfn>PropertyRequestHandler</dfn> callback</h2>
+        <p>Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam nonumy eirmod tempor invidunt ut labore et dolore magna aliquyam.</p>
```
Comment (zolkis) [inline index.html]: I suggest to remove the "Lorem ipsum" stuff, just leave it empty, eventually use "..." or ("TODO") for placeholder, optimally say a short sentence that makes sense for that paragraph.

Comment (danielpeintner) [inline index.html]: Sure. I updated the PR by adding an editorial note instead.

**Comment by zolkis** [comment]
LGTM, merging.

**Pull request merged**